xml/System/Environment.xml

```diff
@@ -608,23 +608,25 @@ Invalid argument
       </Parameters>
       <Docs>
         <param name="message">A message that explains why the process was terminated, or <see langword="null" /> if no explanation is provided.</param>
-        <summary>Immediately terminates a process after writing a message to the Windows Application event log, and then includes the message in error reporting to Microsoft.</summary>
+        <summary>Immediately terminates the process before reporting an error message. For Windows, the error message is written to the Windows Application event log, and the message is included in error reporting to Microsoft. For Unix-like systems, the message, alongside the stack trace, is written to the standard error stream.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
  This method terminates a process without running any active `try`/`finally` blocks or finalizers.
 
- The `Environment.FailFast` method writes the `message` string to the Windows Application event log, creates a dump of your application, and then terminates the current process. The `message` string is also included in error reporting to Microsoft.
+ On Windows, the `Environment.FailFast` method writes the `message` string to the Windows Application event log, creates a dump of your application, and then terminates the current process. The `message` string is also included in error reporting to Microsoft.
+ 
+ Information is reported to Microsoft by using Windows Error Reporting. For more information, see [Windows Error Reporting: Getting Started](/windows-hardware/drivers/dashboard/).
```

Comment:
I would just combine this with the previous paragraph.

Reply:
Done! (35394d55c5592e8ac85f10c951801005b3c9dd1b)